addons/l10n_cl/views/report_invoice.xml

```diff
@@ -1,65 +1,247 @@
+                <span t-field="o.partner_id"
+                      t-options="{&quot;widget&quot;: &quot;contact&quot;, &quot;fields&quot;: [&quot;address&quot;], &quot;no_marker&quot;: true, &quot;no_tag_br&quot;: True}"/>
+
+                <strong>Payment Terms:</strong>
+                <span t-esc="o.invoice_payment_term_id.name or ''"/>
+
+
+                <t t-if="o.invoice_incoterm_id">
+                    <br/>
+                    <strong>Incoterm:</strong>
+                    <span t-field="o.invoice_incoterm_id.name"/>
+                </t>
+
+            </div>
+        </div>
+        <div id="references" class="row">
+            <div name="references" class="col-12 text-center"/>
+        </div>
+    </template>
+
+    <template id="custom_footer">
+        <div name="footer_left_column" class="col-8 text-center"/>
+    </template>
+
     <template id="report_invoice_document" inherit_id="account.report_invoice_document" primary="True">
 
-        <xpath expr="//h2" position="attributes">
-            <attribute name="t-if">not o.l10n_latam_document_type_id.report_name</attribute>
-        </xpath>
+        <t t-set="o" position="after">
+            <t t-set="custom_header" t-value="'l10n_cl.custom_header'"/>
+            <t t-set="report_date" t-value="o.invoice_date"/>
+            <t t-set="report_number" t-value="int(o.l10n_latam_document_number)"/>
+            <t t-set="pre_printed_report" t-value="report_type == 'pdf'"/>
+            <t t-set="report_name" t-value="o.l10n_latam_document_type_id.name"/>
+            <t t-set="header_address" t-value="o.company_id.partner_id"/>
+            <t t-set="is_tax" t-value="o.l10n_latam_document_type_id.code not in ['39', '41', '110', '111', '112', '34']"/>
+            <t t-set="custom_footer">
+                <t t-call="l10n_cl.custom_footer"/>
+            </t>
+        </t>
 
-        <xpath expr="//h2" position="after">
-            <h2 t-if="o.l10n_latam_document_type_id.report_name">
-                <span t-field="o.l10n_latam_document_type_id.report_name"/>
-                <span t-field="o.l10n_latam_document_number"/>
-            </h2>
-        </xpath>
+        <!-- remove default partner address -->
+        <t t-set="address" position="replace"/>
+
+        <xpath expr="//h2" position="replace"/>
 
         <xpath expr="//span[@t-field='line.price_unit']" position="attributes">
             <attribute name="t-field">line.l10n_latam_price_unit</attribute>
         </xpath>
 
-        <xpath expr="//span[@id='line_tax_ids']" position="attributes">
-            <attribute name="t-esc">', '.join(map(lambda x: (x.description or x.name), line.l10n_latam_tax_ids))</attribute>
-        </xpath>
-
         <t t-set="current_subtotal" t-value="current_subtotal + line.price_subtotal" position="attributes">
-            <attribute name="t-value">current_subtotal + line.l10n_latam_price_subtotal</attribute>
+            <!--<t t-if="is_tax">-->
```

Comment:
Need to check what we do with this...

Reply:
by now.. do I remove the comment lines?

Reply:
talked about this in previous session 2020-09-08 it is ok (removed comments)